datasets/lu/administrative_sanctions/crawler.py

```diff
@@ -0,0 +1,97 @@
+from typing import cast
+
+from zavod import Context, helpers as h
+from datetime import datetime
+
+FRENCH_TO_ENGLISH_MONTHS = {
+    "janvier": "january",
+    "février": "february",
+    "mars": "march",
+    "avril": "april",
+    "mai": "may",
+    "juin": "june",
+    "juillet": "july",
+    "août": "august",
+    "septembre": "september",
+    "octobre": "october",
+    "novembre": "november",
+    "décembre": "december",
+}
+
+
+def parse_date(date: str) -> str:
+    for french_month in FRENCH_TO_ENGLISH_MONTHS:
+        date = date.replace(french_month, FRENCH_TO_ENGLISH_MONTHS[french_month])
+
+    try:
+        return datetime.strptime(date, "%d %b %Y")
+    except:
+        return None
```

Comment:
ruff complained about this bare except so I changed it like so
https://github.com/opensanctions/opensanctions/pull/880/commits/a8b232ac774626a6cf7dbcb6d99e0b36a1dbb657
Generally speaking be careful of try...except which doesn't log what happened - expecially ones that aren't very specific about the exceptions they catch because it's easy to hide bugs and make debugging difficult